lib/src/popover.dart

```diff
@@ -96,7 +92,6 @@ Future<T?> showPopover<T extends Object?>({
   double? width,
   double? height,
   VoidCallback? onPop,
-  bool Function()? isParentAlive,
```

Comment:
We should consider using `@Deprecated` here instead of introducing breaking changes.